docs/README.md

```diff
@@ -21,6 +21,8 @@ installed. Also install the following libraries:
     # Following is needed only for generating API docs
     $ sudo pip install sphinx
     $ sudo Rscript -e 'install.packages(c("knitr", "devtools", "roxygen2", "testthat"), repos="http://cran.stat.ucla.edu/")'
+    $ sudo Rscript -e 'install.packages(c("rmarkdown"), repos="http://cran.stat.ucla.edu/")'
+    $ sudo pip install pandoc pandoc-citeproc
```

Comment:
`pandoc` itself is not a python package but it appears the python package `pypandoc` manages it.
Not sure if `pypandoc` works with `pandoc-citeproc` though